server/.env.sample

```
@@ -3,7 +3,7 @@ NODE_ENV=
 PORT=
 
 JWT_SECRET=
-JWT_EXPIRES=
+JWT_EXPIRES=1d
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_EXPIRES key should go before the JWT_SECRET key

(UnorderedKey)